### DISCUSSION TITLE
Detect the timeline when a change in a background occurs

### DISCUSSION THREAD

**Discussion by AhmedEwis** [Breakthrough/PySceneDetect#254] [General]
After several trials with Pyscene, I was not able to develop the ContentDetector to print only the timeline when there is a change in the background only. It detects the timeline when there is a shot change which outputs undesired results. I am wondering if there is a way to detect the timeline when a change in the background only occurs?

**Comment by Breakthrough** [comment]
Do you happen to have an example that you could share?  This might be a good use case for background subtraction, which [DVR-Scan](https://github.com/Breakthrough/DVR-Scan) does, and PySceneDetect v1.0 will be capable of.

**Comment by AhmedEwis** [comment]
I was trying to implement a combination between (ContentDetector + ThresholdDetector) that is similar to the Premiere Adobe Pro strategy in video processing but it was really tough.
After a lot of trials, Pyscene usually gives very close answers to Premiere Adobe Pro, for one movie approximately1500 scenes.
But, if I want to label only with those with the background change only in an original IMDB script (300 scenes) it gonna take a lot of time and iterations and it may not 100% match, how can we deploy Pyscene to detect only those 300 scenes? 
I will check DVR-Scan.
